tools/topology/CMakeLists.txt

```diff
@@ -104,7 +106,8 @@ set(TPLGS
 	"sof-cnl-nocodec\;sof-cnl-nocodec"
 	"sof-cml-rt5682-max98357a\;sof-cml-rt5682-max98357a\;-DPLATFORM=cml"
 	"sof-cml-demux-rt5682-max98357a\;sof-cml-demux-rt5682-max98357a\;-DPLATFORM=cml"
-	"sof-cml-rt1011-rt5682\;sof-cml-rt1011-rt5682\;-DPLATFORM=cml"
+	"sof-cml-rt1011-rt5682\;sof-cml-rt1011-rt5682\;-DPLATFORM=cml\;-DPPROC=volume"
+	"sof-cml-rt1011-rt5682\;sof-cml-eq-rt1011-rt5682\;-DPLATFORM=cml\;-DPPROC=eq-volume"
```

Comment:
what would be eq used for? DMIC or rt1011, can we have some style here about where to pu the eq?

Reply:
This is adding eq pipeline at first playback pipe, which is rt5682. RT1011 is for speaker and pipeline 7. I chose sof-cml-rt1011-rt5682 because it is used in cml helios.